src/main/resources/db/migration/V202210041018.__trigger_II.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE employment_logs (
+    employment_log_id              NUMBER(4) PRIMARY KEY,
+    first_name                     VARCHAR2(30) NOT NULL,
+    last_name                      VARCHAR2(30) NOT NULL,
+    employment_action              VARCHAR2(5) NOT NULL,
+    employment_status_updtd_tmstmp DATE NOT NULL,
+    CONSTRAINT empl_action CHECK ( employment_action IN ( 'HIRED', 'FIRED' ) )
+);
+
+CREATE SEQUENCE logs_seq NOCACHE;
+
+ALTER TABLE employment_logs MODIFY
+    employment_log_id DEFAULT logs_seq.NEXTVAL;
+
+CREATE OR REPLACE PROCEDURE insert_logs (
+    new_first_name                     IN VARCHAR2,
+    new_last_name                      IN VARCHAR2,
+    new_employment_action              IN VARCHAR2,
+    new_employment_status_updtd_tmstmp IN DATE
+) IS
+BEGIN
+    INSERT INTO employment_logs (
+        first_name,
+        last_name,
+        employment_action,
+        employment_status_updtd_tmstmp
+    ) VALUES (
+        new_first_name,
+        new_last_name,
+        new_employment_action,
+        new_employment_status_updtd_tmstmp
+    );
+
+END;
+
+CREATE OR REPLACE TRIGGER hr_migration.insert_log AFTER
+    INSERT OR DELETE ON employees
+    FOR EACH ROW
+BEGIN
+    IF inserting THEN
```

Comment:
`inserting` and `deleting` is a reserved word in sql, so you must write them uppercase